app/src/Homepage/Homepage-components.tsx

```diff
@@ -22,10 +43,19 @@ export const Sidebar: React.FC<SidebarProps> = ( props: SidebarProps ) => {
 
 
 export const LevelTab: React.FC<{ classlevel: string }> = ({ classlevel }) => {   
+    const navigate = useNavigate();
     
     const handleClick = () => {
-        console.log('Button clicked: ' + classlevel);
-        
+        if (classlevel === 'Home') {
+            // If 'Home' button, route to default homepage
+            navigate('/rate-my-cse/');
+        } else {
+            // Else 'CSE X00s' button, split classlevel into an array of
+            // tokens by the " " delimiter and get second element, 'X00s', 
+            // to route to corresponding '/cseX00s' page
```

Comment:
nit: Avoid being too verbose in your comments. You can just a quick high level explaination like "Figures out the correct 'X00s' that corresponds to the tab" (something like that).

Note: nits are like you should change but it you dont have to if you deem them not necessary

Reply:
Changed